src/main/java/org/cloudfoundry/promregator/rewrite/AbstractMetricFamilySamplesEnricher.java

```diff
@@ -48,9 +48,7 @@ public AbstractMetricFamilySamplesEnricher() {
 	}
 	
 	protected abstract List<String> getEnrichedLabelNames(List<String> original);
```

Comment:
Just a minor side note: I would
```suggestion
	protected abstract List<String> getEnrichedLabelNames(List<String> originalNames);
```
to prevent any misunderstand right from the beginning.